lnbits/wallets/breez.py

```diff
@@ -0,0 +1,177 @@
+except ImportError:
+    BREEZ_SDK_INSTALLED = False
+
+if not BREEZ_SDK_INSTALLED:
+
+    class BreezSdkWallet:  # pyright: ignore
+        def __init__(self):
+            raise RuntimeError(
+                "Breez SDK is not installed. "
+                "Ask admin to run `poetry install -E breez` to install it."
+            )
+
+else:
+    import asyncio
+    from pathlib import Path
+    from typing import AsyncGenerator, Optional
+
+    from loguru import logger
+
+    from lnbits import bolt11 as lnbits_bolt11
+    from lnbits.settings import settings
+
+    from .base import (
+        InvoiceResponse,
+        PaymentResponse,
+        PaymentStatus,
+        StatusResponse,
+        Wallet,
+    )
+
+    breez_event_queue: asyncio.Queue = asyncio.Queue()
+
+    class SDKListener(
+        breez_sdk.EventListener  # pyright: ignore[reportUnboundVariable]
+    ):
+        def on_event(self, event):
+            logger.debug(event)
+            breez_event_queue.put_nowait(event)
+
+    class BreezSdkWallet(Wallet):  # type: ignore[no-redef]
+        def __init__(self):
+            if not settings.breez_greenlight_seed:
+                raise ValueError(
+                    "cannot initialize BreezSdkWallet: missing breez_greenlight_seed"
+                )
+            if not settings.breez_api_key:
+                raise ValueError(
+                    "cannot initialize BreezSdkWallet: missing breez_api_key"
+                )
+            if not settings.breez_greenlight_invite_code:
+                raise ValueError(
+                    "cannot initialize BreezSdkWallet: "
+                    "missing breez_greenlight_invite_code"
+                )
+
+            self.config = breez_sdk.default_config(
+                breez_sdk.EnvironmentType.PRODUCTION,
+                settings.breez_api_key,
+                breez_sdk.NodeConfig.GREENLIGHT(
+                    config=breez_sdk.GreenlightNodeConfig(
+                        partner_credentials=None,
+                        invite_code=settings.breez_greenlight_invite_code,
+                    )
+                ),
+            )
+
+            breez_sdk_working_dir = Path(settings.lnbits_data_folder, "breez-sdk")
+            breez_sdk_working_dir.mkdir(parents=True, exist_ok=True)
+            self.config.working_dir = breez_sdk_working_dir.absolute().as_posix()
+
+            seed = breez_sdk.mnemonic_to_seed(settings.breez_greenlight_seed)
+            self.sdk_services = breez_sdk.connect(self.config, seed, SDKListener())
+
+        async def cleanup(self):
+            self.sdk_services.disconnect()
+
+        async def status(self) -> StatusResponse:
+            try:
+                node_info: breez_sdk.NodeState = self.sdk_services.node_info()
+            except Exception as exc:
+                return StatusResponse(f"Failed to connect to breez, got: '{exc}...'", 0)
+
+            return StatusResponse(None, int(node_info.max_payable_msat))
```

Comment:
Better to check the `channels_balance_msat`, which is a sum of the total spendable amount.
```suggestion
            return StatusResponse(None, int(node_info.channels_balance_msat))
```